### ISSUE TITLE
Messages for different conditions when logged in... and Line code 253

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#8059]
@DanielnetoDotCom What does the "Line code 253" error represent. Is there a translation for this code?
This code was displayed when I tried to access a URL directly: https://cnetv.com.br/video/99 (Without the slash at the end of the URL)
When accessing through the full URL with the slash, it works normally: https://cnetv.com.br/video/99/

![image](https://github.com/WWBN/AVideo/assets/37880073/752424ed-3789-455c-a27f-6af6b0267ed8)

**Comment by gbrn** [comment]
@DanielnetoDotCom This problem still appears. Only the error message has changed.
Isn't it correct to work with or without the slash in the URL?

![image](https://github.com/WWBN/AVideo/assets/37880073/0ec6a76f-d5d0-4706-8299-8c4179aa9e18)

**Comment by DanielnetoDotCom** [comment]
what is the URL for this?

**Comment by gbrn** [comment]
> @DanielnetoDotCom What does the "Line code 253" error represent. Is there a translation for this code?
This code was displayed when I tried to access a URL directly: https://cnetv.com.br/video/99 (Without the slash at the end of the URL)
When accessing through the full URL with the slash, it works normally: https://cnetv.com.br/video/99/

Those example URLs I cited:
https://cnetv.com.br/video/99    (Not work)
https://cnetv.com.br/video/99/  (This works)

These URLs give different results.

**Comment by DanielnetoDotCom** [comment]
so add the slash at the end.

**Comment by DanielnetoDotCom** [comment]
please check this update, this should make the URL without slash work

**Comment by gbrn** [comment]
@DanielnetoDotCom Dani, I did the latest pull and the issue still persists.

![image](https://github.com/WWBN/AVideo/assets/37880073/3d4ddf87-0825-4b79-80e0-14341a889c0f)

**Comment by DanielnetoDotCom** [comment]
Please check this update

**Comment by DanielnetoDotCom** [comment]
I guess you need to have something after the last slash

`https://cnetv.com.br/video/99/something`

**Comment by gbrn** [comment]
@DanielnetoDotCom Indeed, placing any character after the slash redirects the link to the correct location.
But.... the use of seed the ID of the video, wouldn't it be to shorten the URL? So it wouldn't be logical to add characters if I want to shorten the URL.
I think this is a BUG...

**Comment by gbrn** [comment]
@DanielnetoDotCom Conseguiu verificar alguma coisa referente a este bug da URL?

**Comment by DanielnetoDotCom** [comment]
I cannot reproduce it on my side

https://tutorials.avideo.com/video/78

**Comment by gbrn** [comment]
@DanielnetoDotCom To access my videos you must be logged in. The URL you are testing does not require a login to access. Check this.

On my side, if I try to access the URL without logging in, it shows this invalid error.
The message should appear that the video is not accessible or not available... Log in...:
https://cnetv.com.br/video/99

![image](https://github.com/WWBN/AVideo/assets/37880073/69f91daf-0c7b-4254-a81d-84cf52a6e328)
